lib/cli/src/commands/login.rs

```diff
@@ -0,0 +1,35 @@
+use clap::Parser;
+use dialoguer::Input;
+
+/// Subcommand for listing packages
+#[derive(Debug, Clone, Parser)]
+pub struct Login {
+    /// Registry to log into (default: wapm.io)
+    #[clap(name = "REGISTRY")]
+    pub registry: Option<String>,
+    /// Login token
+    #[clap(name = "TOKEN")]
+    pub token: Option<String>,
+}
+
+impl Login {
+    fn get_token_or_ask_user(&self) -> Result<String, std::io::Error> {
+        match self.token.as_ref() {
+            Some(s) => Ok(s.clone()),
+            None => Input::new()
+                .with_prompt("Please paste the login token from https://wapm.io/me:\"")
```

Comment:
We might want to use `self.registry` here instead of hard-coding `https://wapm.io/me`.

Reply:
ok done